bip-musig2.mediawiki

```diff
@@ -471,6 +454,64 @@ The reference implementation is for demonstration purposes only and not to be us
 
 == Remarks on Security and Correctness ==
 
+=== Modifications to Nonce Generation ===
+
+Implementors must avoid modifying the ''NonceGen'' algorithm without being fully aware of the implications.
+We provide two modifications that are secure when applied correctly and may be useful in special circumstances, summarized in the following table.
+
+{| class="wikitable" style="margin:auto"
+|-
+!                       !! needs secure randomness !! needs secure counter !! needs to keep state securely !! needs aggregated nonce of all other signers (only possible for one signer)
+|-
+| NonceGen              || ✓      || &nbsp; || ✓      || &nbsp;
```

Comment:
Nit: Should the cells along the left side be headers as well, e.g. in this style: https://www.mediawiki.org/wiki/Help:Tables#Accessibility_of_table_header_cells

Reply:
Good idea, gave it a try.